### ISSUE TITLE
-Xverify:none is deprecated in JDK 13

### ISSUE THREAD

**Issue by schmee** [technomancy/leiningen#2611]
If you run Leininging with an early access build of JDK 13 you are greeted by: 

> OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.

This might lead to some confusion for Leininging users who are not familiar with the JDK, especially for people who are new to both Clojure and Java. 

Here's the relevant JDK bug tracker issue: https://bugs.openjdk.java.net/browse/JDK-8218003

The same issue also states:

> Users who need to run without startup verification can use AppCDS to archive their classes. The classes will get verified during archiving and so avoid verification at runtime. 

Is AppCDS something that could be relevant for Leiningen?

**Comment by g1eny0ung** [comment]
Is there any progress? Same for me in JDK 13.

**Comment by narocath** [comment]
Same here with latest stable leiningen and OpenJDK 13.

**Comment by chinnonsantos** [comment]
Also with me

```
OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
```

with 

```
openjdk 13 2019-09-17
OpenJDK Runtime Environment (build 13+13-Ubuntu-0ubunt1)
OpenJDK 64-Bit Server VM (build 13+13-Ubuntu-0ubunt1, mixed mode, sharing)
```

**Comment by glts** [comment]
For now this can be worked around by clearing the LEIN_JVM_OPTS environment variable before invoking lein:

    export LEIN_JVM_OPTS=

**Comment by carlosefr** [comment]
Won't that disable important things already there as defaults?

**Comment by technomancy** [comment]
The default values there help speed things up, but they're not strictly required.

But this might be better:

export LEIN_JVM_OPTS="-XX:TieredStopAtLevel=1"

It's been a while since I measured the effect of these settings; if you are curious you can try for yourself.

**Comment by technomancy** [comment]
Note to self: the replacement for the deprecated functionality is AppCDS, which is not available in OpenJDK 8, so we can't remove this until after June 2023 when OpenJDK 8 is EOL.

**Comment by samuelhnrq** [comment]
Can't this be smart and automatically switched depending on the java version? Where is this variable set?

**Comment by technomancy** [comment]
No; there is no way to determine the version of Java used until after Java has launched. By then it's too late.

**Comment by samuelhnrq** [comment]
But, isn't this an assumed default value? So some script sets it right? Because if my memory doesn't fail me I remember trying to echo such option before altering and it were unset... So whatever scripts calls `java -jar leningen.jar` sets it right on time doesn't it? I bet it can also check for java version... Like [here](https://github.com/technomancy/leiningen/blob/master/bin/lein#L394)?

**Comment by samuelhnrq** [comment]
I've opened a PR about this, we should move further discussion to there. #2677
